index.js

```diff
@@ -11,20 +11,30 @@
 import ethers from "ethers";
 import Decimal from "decimal.js";
 import * as dotenv from "dotenv";
+import * as fs from "fs";
+import { computeMerkleTree } from "./merkleTree.js";
 
 import dividerAbi from "./abis/Divider.js";
 import tokenAbi from "./abis/Token.js";
 
 dotenv.config();
 
 const DIVIDER_ADDRESS = "0x86bA3E96Be68563E41c2f5769F1AF9fAf758e6E0";
-
-async function main(adapterAddress, startBlock = 0, endBlock = Infinity) {
+const MORPHO_TOKEN = "0x9994E35Db50125E0DF82e4c2dde62496CE330999";
+const PROOFS_FILENAME = "./proof.json";
+
+async function main(
+  adapterAddress = "0x880E5caBB22D24F3E278C4C760e763f239AccA95", // WstETHAdapter
+  startBlock = 0,
+  endBlock = Infinity,
```

Comment:
Could we add some documentation here? :) I'm a bit confused as to what the `startBlock` and `endBlock` parameters signify.

Reply:
yep, good call